### ISSUE TITLE
Is there any config.toml that runs tiflash as a standalone engine?

### ISSUE THREAD

**Issue by nautaa** [pingcap/tiflash#4584]
## General Question

Is there any config.toml that runs tiflash as a standalone engine? If I don't want to run tikv, pd or something for this.

**Comment by flowbehappy** [comment]
Hi, TiFlash currently does not support running without TiKV or PD. BTW, I'm already heard from many places about the similar idea. And I'm very interested in why you guys like TiFlash running in this module. I will be very grateful if you could share your mind?  @nautaa

**Comment by nautaa** [comment]
I just want to learn the design of Tiflash quickly. I found that tiflash can interaction via the tiflash-client cli, is this missing some feature?

**Comment by flowbehappy** [comment]
@nautaa  Yes, `tiflash-client` is still supported currently. It is only retained for debugging purposes and is not supposed to be used in production.
